index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Consul</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#"><img src="./img/consul-logo.png" alt="" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./img/Hamburger_icon.svg.png" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit">
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html" id="current">Home</a></li>
+          <li><a href="about.html">About</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="home-section">
+        <div class="home-wrapper">
+          <div class="home-section-header">
+            <p class="home-greeting">"Hello! Sharing World"</p>
+            <h1 class="home-info">
+              Unlocking the potential of those who advance the world
+            </h1>
+          </div>
+          <div class="home-text">
+            <p>
+              We increase our clients' topline by optimizing their growth
+              strategies, marketing, pricing and sales.
+            </p>
+          </div>
+          <div class="home-timetable">
+            <p>We work from (MON) ~ (SUN)</p>
+          </div>
+          <div class="home-adress">
+            <p>We build a relationship that’s based on trust. Over the years they’ve proven that their delivery is fast and stable</p>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="main-program">
+        <div class="main-program-ul">
+          <h3 class="main-program-intro">Main Program</h3>
+          <p class="main-program-intro-line"></p>
+          <div class="main-grid">
+            <div class="main-program-li">
+              <div class="main-program-img">
+                <img src="./img/i3.png" alt="" />
+              </div>
+              <div class="main-role">
+                <p>Business</p>
+              </div>
+              <div class="main-content">
+                <p>
+                  “If you want to be happy, set a goal that commands your thoughts, liberates your energy and inspires your hopes.”
+                </p>
+              </div>
+            </div>
```

Comment:
- [ ] This section includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.
- The title should be bold,
- The images should have a fixed height, this helps with uniformity,
- Reduce the spacings between each article. 
- Add the `See whole program` subheader below the articles to this section.

_See images below._

**Desktop Screen**

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123959235-c7f9ee80-d962-11eb-82f0-7afa469b4996.png) | ![image](https://user-images.githubusercontent.com/49621279/123977808-2aa7b600-d974-11eb-9166-ee2a9178f888.png)
  
**Mobile Screens**
**Original** | **Your project**
  ----------- | -----------
![image](https://user-images.githubusercontent.com/49621279/123962907-cb8f7480-d966-11eb-8ea6-1258036ddc4d.png) | ![image](https://user-images.githubusercontent.com/49621279/123978622-dcdf7d80-d974-11eb-8003-70d370e1eac5.png)